drivers/filesystems/fastfat/vfat.h

```diff
@@ -17,11 +17,7 @@
 #endif
 
 #define USE_ROS_CC_AND_FS
-#if 0
-#ifndef _MSC_VER
 #define ENABLE_SWAPOUT
```

Comment:
If it works unconditionally, just drop that define.

Reply:
By commenting out the ENABLE_SWAPOUT we get back the old (not dismount) behaviour, but yes correct, once this patch is finalized we can remove all occurrences of ENABLE_SWAPOUT through the code.